crates/sqlexec/src/remote/client.rs

```diff
@@ -199,35 +204,44 @@ impl RemoteClient {
         metadata.insert(DB_NAME_KEY, params.db_name.parse()?);
         metadata.insert(ORG_KEY, params.org.parse()?);
 
+        let mut body = HashMap::new();
+        body.insert("user", params.user);
+        body.insert("password", params.password);
+        body.insert("org_name", params.org);
+        body.insert("db_name", params.db_name);
+        body.insert("api_version", 2.to_string());
+        body.insert("client_type", client_type.to_string());
+
+        info!("client authentication");
```

Comment:
debug?

Reply:
I had it here intentionally actually, but lmk if you think it shouldn't be